gini/gini-switch-sdk-ios
# Feature codable

# Introduction

Switched to Codable for JSON serialization. Converted from a generic Extractions implementation to a concrete set of extractions (e.g. company name, contract name, energy meter number).

# How to test

Run the unit tests. Also use test documents and go through the example app to see that extractions get parse properly. The address should now be split into multiple fields

# Merge info

This should be merge for version 1 since the API for receiving extractions has a breaking change.
